tests/DCPS/Compiler/is_topic_type/run_test.pl

```diff
@@ -76,19 +79,47 @@ sub subtest {
       $status = 1;
     }
   }
-  # Make sure we found everything in %expected
```

Comment:
I think this is an important check and would like to see it stay in the test, but limit it to the filename being checked.

Reply:
Honestly not sure how that got deleted